.mailmap

```diff
@@ -115,11 +115,13 @@ Russell O'Connor <roconnor@blockstream.io>         roconnor-blockstream <roconno
 Christine Paulin <cpaulin@gforge>                  cpaulin <cpaulin@85f007b7-540e-0410-9357-904b9bb8a0f7>
 Christine Paulin <cpaulin@gforge>                  mohring <mohring@85f007b7-540e-0410-9357-904b9bb8a0f7>
 Pierre-Marie Pédrot <pierre-marie.pedrot@inria.fr> ppedrot <ppedrot@85f007b7-540e-0410-9357-904b9bb8a0f7>
+Pierre-Marie Pédrot <pierre-marie.pedrot@inria.fr> Pierre-Marie Pédrot <pierre-marie.pedrot@irif.fr>
 Frederic Peschanski <frederic.peschanski@lip6.fr>  fredokun <frederic.peschanski@lip6.fr>
 Clément Pit-Claudel <clement.pitclaudel@live.com>  Clément Pit--Claudel <clement.pitclaudel@live.com>
 Clément Pit-Claudel <clement.pitclaudel@live.com>  Clément Pit-Claudel <cpitclaudel@users.noreply.github.com>
 Loïc Pottier <pottier@gforge>                      pottier <pottier@85f007b7-540e-0410-9357-904b9bb8a0f7>
 Matthias Puech <puech@gforge>                      puech <puech@85f007b7-540e-0410-9357-904b9bb8a0f7>
+Robert Rand <rnrand@gmail.com>                     Robert Rand <rxtreme@gmail.com>
```

Comment:
@rnrand If you want me to use `rxtreme@gmail.com` rather than `rnrand@gmail.com` for you, let me know

Reply:
I prefer rnrand. (Not sure how the other one got in there...)

Reply:
It's from https://github.com/coq/coq/pull/10563/commits/7865d0baf96ccbde37ed2f85e5ddbd5c24047eba ; is it the email address that gets associated with edits that you make on GitHub itself?

Reply:
Doesn't look like it, but I'll figure it out.